lib/cashbox/model/subscription.rb

```
@@ -30,5 +30,36 @@ class Subscription < Model
     property :policy
     property :starts, coerce: Cashbox::Type.DateTime
     property :status
+
+    def add_subscription_item(product_to_add)
+      subscription_item_to_add = Cashbox::SubscriptionItem.new({
+        product: product_to_add
+      })
+
+      self.items.push(subscription_item_to_add)
+    end
+
+    def remove_subscription_item(product_to_remove)
+      subscription_item_to_remove = nil
+
+      self.items.each do |subscription_item|
+        if subscription_item.product.id == product_to_remove.id
+          subscription_item_to_remove = self.items.delete(subscription_item)
+        end
+      end
+
+      subscription_item_to_remove
+    end
+
+    def replace_subscription_item(product_to_add, product_to_remove)
+      subscription_item_to_remove = self.remove_subscription_item(product_to_remove)
+
+      self.add_subscription_item(product_to_add)
+      self.items.last.replaces = subscription_item_to_remove.product.id if subscription_item_to_remove
```

should we throw an error if the product you want to remove doesn't exist?

---

maybe? enhancement?